web3/shh.py

```diff
@@ -1,21 +1,60 @@
+from web3 import formatters
+from web3.utils.encoding import (
+    to_decimal,
+)
+from web3.utils.functional import (
+    apply_formatters_to_return,
+)
+from web3.utils.filters import (
+    ShhFilter,
+)
+
+
 class Shh(object):
     """
     TODO: flesh this out.
     """
     def __init__(self, web3):
         self.web3 = web3
+        self.request_manager = web3._requestManager
```

Comment:
Instead of storing this, can you either make a property that accesses this or access it directly through the `web3` object.  This pattern will break some of the new managers that I've been working on that distribute requests across multiple providers or managers.

```python
@property
def request_manager(self):
    return self.web3._requestManager
```